### PR TITLE
feat(shared): add helper to convert string to RTE value

### PR DESCRIPTION
**Pull request by celine-s**

### CODE REVIEW

File: packages/asset-kit-block/src/components/InformationSection.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { hasRichTextValue, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import { convertToRTEValue, hasRichTextValue, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
```
Comment (Walkeezy) [inline packages/asset-kit-block/src/components/InformationSection.tsx:3]: `convertToRTEValue` should be `convertToRteValue`, see: https://weare.frontify.com/document/42#/frontend-react/feature-file-structure 😬

Comment (celine-s) [reply packages/asset-kit-block/src/components/InformationSection.tsx]: Thanks! Adapted

File: packages/shared/src/helpers/convertToRichTextValue.spec.ts
Comment (ragi96) [inline packages/shared/src/helpers/convertToRichTextValue.spec.ts:1]: add a test with align setted

File: packages/audio-block/src/AudioBlock.tsx [hunk 2]
```diff
@@ -102,8 +103,8 @@ export const AudioBlock = ({ appBridge }: BlockProps) => {
     };
 
     const updateAudioAsset = async (audio: Asset) => {
-        if (!title) {
-            saveTitle(`[{"type":"heading3","children":[{"text":"${audio.title}"}]}]`);
+        if (!hasRichTextValue(title)) {
+            saveTitle(convertToRteValue(audio?.title, TextStyles.ELEMENT_HEADING3));
```
Comment (ragi96) [inline packages/audio-block/src/AudioBlock.tsx:107]:
```suggestion
            saveTitle(convertToRteValue(audio.title, TextStyles.ELEMENT_HEADING3));
```

File: packages/shared/src/helpers/convertToRichTextValue.spec.ts
Comment (celine-s) [reply packages/shared/src/helpers/convertToRichTextValue.spec.ts:1]: there is one on line 21. what for a test do you think of?

Comment (ragi96) [reply packages/shared/src/helpers/convertToRichTextValue.spec.ts:1]: sorry, short time blindness hit me

File: packages/shared/src/helpers/convertToRichTextValue.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { TextStyles } from '@frontify/fondue';
+import { hasRichTextValue } from './hasRichTextValue';
+
+export const convertToRteValue = (
+    text: string,
+    textStyle: TextStyles,
+    align?: 'center' | 'right' | 'left' | 'justify'
+) => (hasRichTextValue(text) ? text : JSON.stringify([{ type: textStyle, children: [{ text: text ?? '' }], align }]));
```
[the remark this answers is not in the document]
Comment (celine-s) [reply packages/shared/src/helpers/convertToRichTextValue.ts]: You were right. I now enabled that text can be undefined. This makes it easier to reuse the function for empty RTE values

**Review by ragi96 — APPROVED** [review]

File: packages/shared/src/helpers/convertToRichTextValue.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/shared/src/helpers/convertToRichTextValue.ts]: Oh yeah thats even better

**Review by Walkeezy — APPROVED** [review]

**Pull request merged**